akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -344,6 +346,7 @@ object ClusterClient {
  * Note that this is a best effort implementation: messages can always be lost due to the distributed
  * nature of the actors involved.
  */
+@deprecated("Use Akka gRPC instead.", since = "2.6.0")
```

Comment:
Is this the only place we'd need it though, on the object it of course breaks the build.

Reply:
Is that ever used by users, though? It seems we should deprecate the things that users use, i.e. that are shown on the docs page. I'd suggest these:

 * `object ClusterClient` and/or `ClusterClient.props`
 * `object ClusterClientReceptionist`